tests/interactive_test.py

```diff
@@ -0,0 +1,270 @@
+################################################################################
+# Interactive Tests
```

Comment:
One thing we might do is adding an interactive testing here:
https://github.com/dcmoura/spyql/blob/c379afa89c883b29bc4d3eaf3eea5a7c9d9cbd91/tests/main_test.py#L67

This way, all tests done to the CLI would be also done in interactive mode. Then, we would only need to tackle here features that are specific to the interactive mode.

I will skip this file for this review and come back in the next one.